workshop/README.md

```diff
@@ -1,9 +1,31 @@
-# Istio 101 workshop
+# Advanced Istio course (name TBD)
+[Istio](https://www.ibm.com/cloud/info/istio) is an open platform to connect, secure, and manage a network of microservices, also known as a service mesh, on cloud platforms such as Kubernetes in IBM Cloud Container Service. With Istio, manage network traffic, load balance across microservices, enforce access policies, verify service identity on the service mesh, and more.
 
-## Workshop Setup
-- [Exercise 1 - Accessing a Kubernetes cluster with IBM Cloud Container Service](exercise-1/README.md)
+In this course, you can see how to install Istio alongside microservices for a simple mock app called Guestbook. The microservices include a product web page, details, reviews, and ratings. When you deploy Guestbook's microservices into an IBM Cloud Container Service cluster where Istio is installed, you inject the Istio Envoy sidecar proxies in the pods of each microservice.
```

Comment:
> The microservices include a product web page, details, reviews, and ratings.

This is not accurate.  The guest book doesn't contain details/reviews/rating or product page.

Reply:
I'll update.